src/MICore/Debugger.cs

```diff
@@ -1172,6 +1172,11 @@ private void HandleThreadGroupExited(Results results)
             if (isThreadGroupEmpty)
             {
                 ScheduleStdOutProcessing(@"*stopped,reason=""exited""");
+
+                // Processing the fake "stopped" event sent above will normally cause the debugger to close, but if
+                //  the debugger process is already gone (e.g. because the terminal window was closed), we won't get
+                //  a response, so queue a fake "exit" event for processing as well, just to be sure.
+                ScheduleStdOutProcessing("^exit");
```

Comment:
Did we verify that this isn't accidently causing the stop code path to occur twice? Also, we should check if this works okay on clrdbg and against android. Adding @gregg-miskelly and @chuckries